patchwork-extensions-block/src/main/resources/patchwork-extensions-block-harvest.mixins.json

```diff
@@ -0,0 +1,17 @@
+{
+  "required": true,
+  "package": "net.patchworkmc.mixin.extensions.block.harvest",
+  "compatibilityLevel": "JAVA_8",
+  "mixins": [
+    "MixinOreBlock",
```

Comment:
Up to you on if you want to remove this extra json file too.

Reply:
I think we should make separate mixins, otherwise some mixins will get really long. I would like to have separate mixin.json, it is more modular.